office_space_allocation/amity.py

```diff
@@ -0,0 +1,150 @@
+import random
+
+from office_space_allocation.fellow import Fellow
+from office_space_allocation.office import Office
+from office_space_allocation.person import Person
+from office_space_allocation.staff import Staff
+from office_space_allocation.utilities import InvalidRoomOccupantError, RoomFullError, MultiplePeopleFoundException
+
+
+class Amity:
+    def __init__(self):
+        self.all_persons = []
+        self.all_rooms = []
+        self.allocated_rooms = []
+
+    def add_person(self, person):
+        """
+        Adds a new Person to list of persons
+        :param: ```Person``` person
+        """
+        if not isinstance(person, Person):
+            raise TypeError("Argument should be of type Person")
+        else:
+            self.all_persons.append(person)
+
+    def add_room(self, new_room):
+        """
+        Adds ```Room``` new_room to list of all_rooms
+        :param ```Room``` new_room
+        """
+        self.all_rooms.append(new_room)
+
+    def has_room(self, room):
+        """
+        Checks if Room `room` already exists in Amity
+        :return: `True` if room exist, otherwise `False`
+        """
+        for r in self.all_rooms:
+            if r == room:
+                return True
+        return False
+
```

Comment:
How about `if room in self.all_rooms:` It is shorter and more pythonic than `for r in self.all_rooms:
 +            if r == room:
 +                return True
 +        return False`